### PR TITLE
Error in the API routing

### PR DESCRIPTION
**Pull request by matrix-amiel**
Hi @bradymiller 

We contributed a nice feature last week that enables us to extend the Rest API using Events (#2804).
Probably we missing important lines and the API calls failed now. The code loads the _rest_routes.inc.php file before globas.php is loaded and the autoload of composer still not exist, without autoload the class RestApiCreateEvent not found and we got an error...

In this fix, I move the require of the _rest_routes.inc.php after the loaded of globals.php. I tested it as best I can and I hope all works well now.

Thanks
Amiel

### CODE REVIEW

File: apis/dispatch.php [hunk 1]
```diff
@@ -77,6 +77,8 @@
 
 require_once("./../interface/globals.php");
 require_once("./../library/acl.inc");
+// Include our routes and init routes global
+require_once("./../_rest_routes.inc.php");
```
Comment (sjpadgett) [inline apis/dispatch.php]: Routes needs to be available regardless of authentication. If you need to ensure classes are loaded, do so in _rest_routes by running autoloader there.

**Comment by matrix-amiel** [comment]
Hi @sjpadgett 
Ok. I agree with your comment... I pushed another solution, I think it's cleaner. 
It's not enough to run autoload, we need `$GLOBALS["kernel"]`  for the events system.

**Comment by sjpadgett** [comment]
Hi @matrix-amiel 
I think you're right. Much cleaner. I'll do some testing later tonight and let you know.
Thanks.

**Comment by sjpadgett** [comment]
Hi @matrix-amiel , Thanks for the changes and testing okay against my concerns so, i'm good to go.
Not sure what to do about travis @bradymiller , will you merge for me if ravis is okay?

**Comment by stephenwaite** [comment]
hi @sjpadgett since travis is already broken by my previous commit probably easiest to fix after bring in all of the bs4 changes so as not to create any unnecessary conflicts just in case

**Comment by sjpadgett** [comment]
Thanks @stephenwaite Agreed on BS4 but this will go to master core. Still these are caused by minow space on some closures and would pass anyway so bringing in so Matrix guys can do their thing. We all gotta a thing don't ya know!
Unless you object?

**Comment by bradymiller** [comment]
hi @sjpadgett and @stephenwaite ,
Just ignore travis for now; ie. a big red X is good now :)
(when we get the bootstrap code into codebase, then will do a quick travis cleanup)
Agree to bring this PR in master
-brady

**Pull request merged**